app/src/config/development.toml

```
@@ -0,0 +1,63 @@
+[docker]
+host = "localhost"
+
+[core]
+daiTokenContractAddress = "0x642d2b84a32a9a92fec78ceaa9488388b3704898"
+dataUnionFactorySidechainCreationBlock = 0
+dataUnionPublishMemberLimit = 1
+marketplaceContractAddress = "0xF1371c0f40528406dc4f4cAf89924eA9Da49E866"
+platformOriginUrl = ""
+streamrUrl = ""
+uniswapAdaptorContractAddress = "0xE4eA76e830a659282368cA2e7E4d18C4AE52D8B3"
+web3TransactionConfirmationBlocks = 1
+
+[[core.storageNodes]]
+name = "Local (broker-node-storage-1)"
+address = "0xde1112f631486CfC759A50196853011528bC5FA0"
+
+[client]
+graphUrl = ":8000/subgraphs/name/streamr-dev/network-contracts"
+restUrl = "/api/v1"
```

This will become `/api/v2` with the Brubeck API backend (still in [branch](https://github.com/streamr-dev/core-api/pull/1150)). It's not yet in docker, so you might not want to make this change yet.

```suggestion
restUrl = "/api/v2"
```